lib/ansible/cli/galaxy.py

```diff
@@ -237,7 +237,7 @@ def add_init_options(self, parser, parents=None):
                                  help='The path in which the skeleton {0} will be created. The default is the '
                                       'current working directory.'.format(galaxy_type))
         init_parser.add_argument('--{0}-skeleton'.format(galaxy_type), dest='{0}_skeleton'.format(galaxy_type),
-                                 default=C.GALAXY_ROLE_SKELETON,
+                                 default=C.GALAXY_ROLE_SKELETON if galaxy_type == 'role' else None,
```

Comment:
The name and description should be updated here for the role default too https://github.com/ansible/ansible/blob/devel/lib/ansible/config/base.yml#L1451

Reply:
Oh, it looks like GALAXY_ROLE_SKELETON_IGNORE is also used for collections https://github.com/ansible/ansible/blob/devel/lib/ansible/config/base.yml#L1459